.gitignore

```diff
@@ -3,3 +3,4 @@ lib/
 __fixtures_build__/
 src/__fixtures__/dist/
 coverage/
+yarn-error.log
```

Comment:
Don't put it here, use your global `.gitignore` to ignore all `*.log`.

Reply:
I didn't know that there was a global gitignore 😮 ! I will do it :)